src/styles/theme.ts

```diff
@@ -0,0 +1,13 @@
+export const theme = {
+  colors: {
+    white: '#ffffff',
+    orange: {
+      400: '#ee8328',
+      500: '#e14318',
+      600: '#b7410e',
+    },
+    gray: {
+      100: '#ececec',
+    },
+  },
+};
```

Comment:
Pode colocar um `as const` no final para o Typescript entender que essa variavel tema é apenas Readonly, assim melhora o autocomplete.

https://mainawycliffe.dev/blog/const-assertion-in-typescript/

Reply:
Arrumei a autocomplite, mas de outra forma, desse jeito não estava funcionando